deployment/helm/values.yaml

```diff
@@ -164,6 +164,45 @@ onvifVideo:
     # protocol is the service protocol of the instance service
     protocol: TCP
 
+udev:
+  # enabled defines whether to load a udev configuration
+  enabled: false
+  # name is the Kubernetes resource name that will be created for this
+  # udev configuration
+  name: akri-udev
+  # properties is a map of properties that will be passed to any instances
+  # created as a result of applying this udev configuration
+  properties:
+  # udevRules is the list of udev rules used to find instances created as a result of
+  # applying this udev configuration
+  udevRules:
```

Comment:
not sure if it is needed, but I've initialized with empty array in other places: `udevRules: []`

Reply:
I wanted it to throw an error if empty so it would prompt people to add a rule when trying to install helm, but I can see why setting an empty array would be good.

Reply:
I like your idea more than initializing with []